.github/workflows/container-tests.yml

```diff
@@ -3,7 +3,56 @@ on:
     types:
       - created
 jobs:
+  distgen-check:
+    name: "Check distgen generated files"
+    runs-on: ubuntu-20.04
+    if: |
+      github.event.issue.pull_request
+      && (contains(github.event.comment.body, '[test]') || contains(github.event.comment.body, '[test-all]'))
+      && contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: "refs/pull/${{ github.event.issue.number }}/head"
+          submodules: true
+
+      - name: Check distgen generated files
+        id: check
+        shell: bash
+        run: |
+          sudo apt update && sudo apt -y install python3-pip
+          pip3 install pyyaml distgen Jinja2
```

Comment:
distgen depends on pyyaml and Jinja2 so why do we need to list them here?

Reply:
Hmm, I have added Jinja2, because distgen generation has failed without explicitly specifying it...
But, as you say it, I am not sure why this happened...
However I would rather leave it here explicitly, if you do not mind, as it does not break anything.